### PR TITLE
Update CoC to best exemplify Notre Dame Values

### PR DESCRIPTION
**Pull request by ghost**
Please see above.

### CODE REVIEW

**Comment by AndroidKitKat** [comment]
what happened to the good CoC

File: Code_of_Conduct.md [hunk 1]
```diff
@@ -42,4 +42,89 @@ If you think the tangent is an important and pertinent issue, please bring it up
...
+54. Speak no useless words or words that move to laughter.
+55. Do not love much or boisterous laughter.
+56. Listen willingly to holy reading.
+57. Devote yourself frequently to prayer.
+58. Daily in your prayers, with tears and sighs, confess your past sins to God, and amend them for the future.
+59. Fulfill not the desires of the flesh; hate your own will.
+60. Obey in all things the commands of those whom God has placed in authority over you even though they (which God forbid) should act otherwise, mindful of the Lord's precept, "Do what they say, but not what they do."
+61. Do not wish to be called holy before one is holy; but first to be holy, that you may be truly so called.
+62. Fulfill God's commandments daily in your deeds.
+63. Love chastity.
```
Comment (sambattalio) [inline Code_of_Conduct.md:123]: I like this one specifically

**Comment by AndroidKitKat** [comment]
I lied

**Comment by AndroidKitKat** [comment]
is that breaking part of the coc?

File: Code_of_Conduct.md [hunk 2]
```diff
@@ -42,4 +42,89 @@ If you think the tangent is an important and pertinent issue, please bring it up
 
 By making an effort to follow these guidelines, we will encourage more contribution to our projects, and our discussions will be friendlier and reach conclusions more easily.
 
+## Instruments of Good Works
```
Comment (sambattalio) [inline Code_of_Conduct.md:49]: Can you add something signifying this is an addition to the CoC and not original?

**Comment by ghost** [comment]
@AndroidKitKat The "Good" CoC is still there. I would never be so brash to as delete an entire CoC.

File: Code_of_Conduct.md (hunk 2 above)
Comment (AndroidKitKat) [reply Code_of_Conduct.md:49]: I concur

**Review by AndroidKitKat — APPROVED** [review]
yeah I like it

**Comment by AndroidKitKat** [comment]
send it yeet

File: Code_of_Conduct.md (hunk 2 above)
Comment (ghost) [reply Code_of_Conduct.md:49]: I have made a change with the most recent commit.

**Pull request merged**